superset-frontend/plugins/plugin-chart-echarts/src/Pie/controlPanel.tsx

```diff
@@ -87,6 +88,23 @@ const config: ControlPanelConfig = {
             },
           },
         ],
+        [
+          {
+            name: 'roseType',
+            config: {
+              type: 'SelectControl',
+              label: t('Rose Type'),
+              default: roseType,
+              renderTrigger: true,
+              choices: [
+                ['area', t('Area')],
+                ['radius', t('Radius')],
+                [false, t('None')],
```

Comment:
```suggestion
                [null, t('None')],
```

Reply:
I think it's better to keep the value as false; otherwise, the select control will not behave correctly, and we will never get a chance to choose the None option.

Reply:
@hexcafe `null` [is a valid value](https://github.com/search?q=repo%3Aapache%2Fsuperset++%5Bnull%2C+t%28%27None%27%29%5D%2C&type=code) for the select control. If you prefer to use `False`, then you need to change the Typescript definition accordingly as it's allowing `undefined`.